extensions/opentelemetry/opentelemetry/runtime/pom.xml

```diff
@@ -44,6 +44,16 @@
             <groupId>io.opentelemetry</groupId>
             <artifactId>opentelemetry-sdk</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
Should we exclude all transitives and rely on the `opentelemetry-sdk` dependency to bring in the version we want?

I know Maven "should" know the right thing to do, but wondering if we need to be more explicit?